test/utils.py

```diff
@@ -150,24 +153,25 @@ def script(self) -> Script:
         script.script = "import os\nprint('foo')\n"
         return script
 
-    def insertable_reduction(self, instrument: Instrument) -> Reduction:
+    def insertable_job(self, instrument: Instrument) -> Job:
         """
         Given an instrument model, generate random; reduction, run, and script all related.
```

Comment:
Docstring is now very outdated

Reply:
I don't think this is out of date. What do you mean by this?

Reply:
It says

> Given an instrument model, generate random; reduction, run, and script all related.

Reply:
Ah yes.